### PR TITLE
stopped dealing in absolutes

### LINKED ISSUE #1: Only a Sith deals in absolutes
Only a Sith deals in absolutes.

There is more than one answer to the meaning of life and everything. This codebase could stand to contain less heresy.

### PR DESCRIPTION
**Pull request by Kapparina**
Fixes #1

Updated the project with a 50/50 chance to have the correct answer selected; there are now 3 correct answers and 3 incorrect answers.

### CODE REVIEW

**Comment by Kapparina** [comment]
I showed you my PR, please answer me.

File: src/main.rs [hunk 1]
```diff
@@ -1,20 +1,49 @@
...
+
 fn main() {
-    println!("Hello, world!");
-    println!("Hello, world!");
+    let now = SystemTime::now();
+    match now.duration_since(UNIX_EPOCH) {
+        Ok(duration) => {
+            let seconds_since = duration.as_secs();
+            if seconds_since % 5 == 0 {
+                println!("UWUUUUUUUUUUUUUUUUUUUUU");
```
Comment (MrDwarf7) [inline src/main.rs]: Need a better message for when the modulo isn't valid

File: src/main.rs [hunk 2]
```diff
@@ -1,20 +1,49 @@
...
+            } else {
+                println!("YOU HAVE COMMITTED HERESY!");
+            }
+        }
+        Err(_) => {
```
Comment (MrDwarf7) [inline src/main.rs]: Implement a custom Error enum that impls Error from the std to handle this, and remove the direct error call in favor of a question mark

File: src/main.rs [hunk 3]
```diff
@@ -1,20 +1,49 @@
...
-    if question == "What is the answer to life, the universe, and everything?" {
-        return base.parse().unwrap();
+    if question == "What is the answer to life, the universe, and everything in it?" {
+        return draw_answer_from_hat();
     }
     0
 }
+
+fn draw_answer_from_hat() -> i32 {
+    let numbers = [1, 2, 3, 42, 69, 9001];
```
Comment (MrDwarf7) [inline src/main.rs]: Array is missing 420

File: src/main.rs [hunk 4]
```diff
@@ -1,20 +1,49 @@
...
+    let mut rng = thread_rng();
```
Comment (MrDwarf7) [inline src/main.rs:59]: Please use xoshiro rng generators for this project

**Review by MrDwarf7 — CHANGES_REQUESTED** [review]
Once issues have been resolved this can be merged into main.
If any assistance please follow up

File: src/main.rs (hunk 1 above)
Comment (Kapparina) [reply src/main.rs]: Surely you're a fan of Doctor Strange?

File: src/main.rs (hunk 4 above)
Comment (Kapparina) [reply src/main.rs:59]: That sounds like an absolute, only a Sith deals in absolutes.

File: src/main.rs (hunk 2 above)
Comment (Kapparina) [reply src/main.rs]: Job done

**Review by MrDwarf7 — APPROVED** [review]

**Comment by MrDwarf7** [comment]
lgtm

**Pull request merged**